src/Provider.tsx

```diff
@@ -1,48 +1,47 @@
-import React, { FunctionComponent, useEffect, useMemo, useState } from 'react'
+import React, { FunctionComponent, useEffect, useRef, useState } from 'react'
 import eq from 'deep-equal'
 import StreamrClient from 'streamr-client'
 import ClientContext from './ClientContext'
+import type { StreamrClientConfig } from 'streamr-client'
 
-type Props = {
-    children: string,
-    autoConnect?: boolean,
-    autoDisconnect?: boolean,
+export type Props = {
+    children: React.ReactNode,
+} & StreamrClientConfig
+
+const isSSR = typeof window !== 'undefined'
```

Comment:
Should be `===` instead of `!==`

This will break the tests.. is the `StreamrClient` actually dependent of `window` or why do we need this check?